README.md

```diff
@@ -8,27 +8,30 @@ This is a prototype of UMass Transit's incident tracking model, designed for use
 
 ## Development
 
-After bundling and database setup, you can obtain data either by seeding:
-
-```
-rails db:seed
-```
-
-Or by importing XML files from Hastus:
+### Setup
+1. `bundle`
+2. Create the database: `rails db:create`
+3. Run migrations: `rails db:migrate`
+4. Create development data: `rails db:seed`
 
+Alternativley you can import your data from Hatus:
```

Comment:
I'm not sure if this works at the moment.

Reply:
You spelled Alternatively wrong

Reply:
Listen, my spell checker stopped spell checking for some reason and we all know I can't spell on my own.